ViAn/Video/shapes/shape.h

```diff
@@ -20,7 +20,8 @@ class Shape {
     static cv::Scalar qcolor_to_scalar(QColor col);
     static cv::Point qpoint_to_point(QPoint pnt);
 
-    static const int LINE_THICKNESS = 2;
+    static const int LINE_THICKNESS = 2; // Constant used for the thickness of the drawn shapes.
+    static constexpr double ALPHA = 0.6; // Costant used for the opacity.
 
 protected:
     cv::Scalar colour;
```

Comment:
Nice.
I just commited the other changes. I fixed te variable name. And I added the checkbox for include/exclude. And I added a circle on the last choosen point to indocate it.